it/controllers/propertyLinking/UploadControllerISpec.scala

```diff
@@ -0,0 +1,1417 @@
+  val evidenceTextWelsh = "Tystiolaeth"
+  val removeFileTextWelsh = "Dileu"
+  val errorTextWelsh = "Gwall:"
+  val errorSummaryTextWelsh = "Mae yna broblem"
+  val errorMessageTextWelsh = "Dewis bil ardrethi"
+
+  // Locators
+  val headingLocator = "h1"
+  val captionLocator = "#main-content > div > div > span"
+  val backLinkLocator = "#back-link"
+  val localCouncilReferenceLocator = "#local-authority-reference"
+  val propertyLocator = "#address"
+  val hintTextLocator = "#fileTypesText"
+  val fileUploadComponentLocator = "#newFile"
+  val fileUploadButtonLocator = "#newFileButton"
+  val evidenceLocator = "#main-content > div > div > dl > div > dt"
+  val fileNameLocator = "#main-content > div > div > dl > div > dd.govuk-summary-list__value"
+  val removeFileLocator = "#remove-file"
+  val continueButtonLocator = "#continue"
+  val errorSummaryTitleLocator = "#error-summary-heading"
+  val errorSummaryMessageLocator = "#main-content > div > div > div.govuk-error-summary > div > ul > li > a"
+  val errorMessageLocator = "#file-upload-1-error"
+
+  val otherEvidenceHintLocator = "#evidenceType-hint"
+  val otherEvidenceRadioLocator: Int => String = (radioNumber: Int) => s"#upload-evidence-options > div:nth-child($radioNumber) > input"
+  val otherEvidenceRadioLabelLocator: Int => String = (radioNumber: Int) => s"#upload-evidence-options > div:nth-child($radioNumber) > label"
+  val otherEvidenceSummaryLocator = "#main-content > div > div > div.govuk-error-summary > div > h2"
+  val otherEvidenceSummaryMessageLocator = "#main-content > div > div > div.govuk-error-summary > div > div > ul > li > a"
+  val otherEvidenceErrorMessageLocator = "#evidenceType-error"
+  val orLocator = "#upload-evidence-options > div.govuk-radios__divider"
+
+  val cannotProvideEvidenceP1Locator = "#main-content > div > div > p:nth-child(4)"
+  val cannotProvideEvidenceP2Locator = "#main-content > div > div > p:nth-child(5)"
+  val cannotProvideEvidenceP3Locator = "#main-content > div > div > p:nth-child(6)"
+  val cannotProvideEvidenceEmailLocator = "#main-content > div > div > p:nth-child(6) > a"
+
+  // Links
+  val emailHref = "mailto:ccaservice@voa.gov.uk"
+  val backToEvidenceQuestionHref = "/business-rates-property-linking/my-organisation/claim/property-links/evidence"
+  val backToOtherEvidenceHref = "/business-rates-property-linking/my-organisation/claim/property-links/evidence/OTHER/upload"
+  val errorMessageHref = "#newFileButton"
+  val otherEvidenceErrorMessageHref = "#evidenceType"
+  val evidenceTypeRedirectUrl: String => String = (evidenceType: String) => s"/business-rates-property-linking/my-organisation/claim/property-links/evidence/$evidenceType/upload"
+  val checkYourAnswersRedirectUrl = "/business-rates-property-linking/my-organisation/claim/property-links/summary"
+
+  "UploadController show method" should {
+
+    // IP - English
+    "display the correct content in English for an IP that is uploading a business rates bill" which {
```

Comment:
might be worth putting the relationship type in the name of these tests

Reply:
On it 👍